sdk/python/packages/flet/src/flet/controls/theme.py

```diff
@@ -2073,12 +2074,6 @@ class DatePickerTheme:
     the shape decoration of the day label.
     """
 
-    locale: Optional[Locale] = None
-    """
-    An optional locale argument can be used to set the locale for the date picker. It
-    defaults to the ambient locale provided by Localizations.
-    """
-
```

Comment:
Any reason for this change?

Reply:
We currently don't have `locale` in DatePicker control, so it's being configured for the whole page via `page`.